OracleEDQ/README.md

```diff
@@ -0,0 +1,120 @@
+
+For more information about Oracle Enterprise Data Quality please see the [Oracle Enterprise Data Quality 12.2.1.3.0 Online Documentation]( http://www.oracle.com/technetwork/middleware/oedq/documentation/index.html).
+
+The certification of Oracle Enterprise Data Quality on Docker does not require the use of any file presented in this repository. Customers and users are welcome to use them as starters, and customize/tweak, or create from scratch new scripts and Dockerfiles.
+
+For pre-built images containing Oracle software, please check the [Oracle Container Registry](https://container-registry.oracle.com).
+
+## Database prerequisite
+
+You need to have a running Oracle Database, either in a Docker container or on a host. 
+The database connection details are required for creating midtier schemas for use by the EDQ domain.  The schemas are created automatically when the EDQ container is started.
+ 
+If using a 12.1.0.2 CDB/PDB database, ensure PDB is used when creating the schemas. CDB is not supported.
+
+An Oracle Database 12.1.0.2 container can be created from [OracleDatabase](https://github.com/oracle/docker-images/tree/master/OracleDatabase/dockerfiles/12.1.0.2). 
+Follow the instructions to create a 12.1.0.2-based Enterprise Edition database.
+
+Default installs of Oracle Database 12.2 are not supported - please see [Known Issues](#known-issues).
+
+## Oracle Enterprise Data Quality Docker Image Creation
+
+To build a EDQ image you start by building Oracle JDK and Oracle FMW Infrastructure images.
+
+### Building the Oracle JDK (Server JRE) Image
+
+Download the Oracle Server JRE binary into folder `OracleJDK/java-8` and build the image:
+
+        $ cd OracleJDK/java-8
+        $ docker build -t oracle/serverjre:8 .
+
+Please refer to README.md under [OracleJava](https://github.com/oracle/docker-images/tree/master/OracleJava) for details on how to build Oracle Java image.
+
+### Building the Oracle FMW Infrastructure Image
+ 
+Download the binary of FMW Infrastructure into folder `OracleFMWInfrastructure/dockerfiles/12.2.1.3` and build the 12.2.1.3 infrastructure image:
+
+        $ cd OracleFMWInfrastructure/dockerfiles
+        $ ./buildDockerImage.sh -v 12.2.1.3
+
+Please refer to README.md under [OracleFMWInfrastructure](https://github.com/oracle/docker-images/tree/master/OracleFMWInfrastructure) for details on how to build FMW Infrastructure image.
+
+### Building the Oracle Enterprise Data Quality Image
+
+Download the two binaries for [Oracle Enterprise Data Quality 12.2.1.3.0]( http://www.oracle.com/technetwork/middleware/oedq/downloads/index.html) for Linux x86-64-bit into folder `OEDQ/dockerfiles/12.2.1.3.0`.
+
+If a proxy is needed for the host to access yum.oracle.com during build, then first set up the appropriate environment, e.g.:
+
+        $ export http_proxy=myproxy.example.com:80
+        $ export https_proxy=myproxy.example.com:80
+        $ export no_proxy="localhost,127.0.0.1,localaddress,.localdomain.com"
```

Comment:
I'm not sure why you're including `localaddress` and `.localdomain.com` in the `no_proxy` list.

Reply:
Removed exporting no_proxy